sampleApp/sample.tsx

```diff
@@ -36,28 +46,39 @@ oneNoteDataProvider.getNotebooks().then((notebooks) => {
 			notebookListUpdater: updater,
 			callbacks: {
 				onNotebookHierarchyUpdated: (newNotebookHierarchy) => {
+					let latestNotebook = newNotebookHierarchy[newNotebookHierarchy.length - 1].name;
+					if(defaultDropdownLabel === "")
+						defaultDropdownLabel = latestNotebook; 
 					render(globalProps, newNotebookHierarchy);
+					renderDropdown(globalProps, newNotebookHierarchy, defaultDropdownLabel, true);
 				},
 				onSectionSelected: (section, breadcrumbs) => {
 					globalProps.globals.selectedId = section.id;
-
+					defaultDropdownLabel = section.name; 
 					// tslint:disable-next-line:no-console
 					console.log(breadcrumbs.map(x => x.name).join(' > '));
 
 					render(globalProps, globalProps.globals.notebookListUpdater!.get());
+					renderDropdown(globalProps, globalProps.globals.notebookListUpdater!.get(),  defaultDropdownLabel, false);
 				},
 				onPageSelected: (page, breadcrumbs) => {
 					globalProps.globals.selectedId = page.id;
-
+					defaultDropdownLabel = page.name;
 					// tslint:disable-next-line:no-console
 					console.log(breadcrumbs.map(x => x.name).join(' > '));
+					console.log(breadcrumbs[breadcrumbs.length -1].name)
 
 					render(globalProps, globalProps.globals.notebookListUpdater!.get());
+					renderDropdown(globalProps, globalProps.globals.notebookListUpdater!.get(), defaultDropdownLabel, false);
 				},
 				onAccessibleSelection: (selectedItemId: string) => {
```

Comment:
onAccessibleSelection does not always mean the item was clicked on, e.g., the item might have been navigated to via arrow keys, and so while it's focused on, it's not the new selected item. We should not change the label in this case (we delegate that responsibility to onSection/PageSelected.